tests/phpunit/tests/customize/selective-refresh-ajax.php

```diff
@@ -188,8 +188,6 @@ public function test_handle_render_partials_request_for_non_rendering_partial()
 			)
 		);
 
-		$count_customize_render_partials_before = has_action( 'customize_render_partials_before' );
-		$count_customize_render_partials_after  = has_action( 'customize_render_partials_after' );
```

Comment:
I think these tests adding 1 were working, but not checking what was originally intended. I think these changes are correct. But wanted to note what was going on for future reference.

By default, Core does not attach anything to `customize_render_partials_before` or `customize_render_partials_after`. So `has_action()` was returning `false`. Performing `false + 1` below changed the value to `1`, and then it was being loosely compared to `true` (and passing).